PhotoCollection/PhotoCollection/View Controllers/PhotosCollectionViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  PhotosCollectionViewController.swift
+//  PhotoCollection
+//
+//  Created by Nathanael Youngren on 1/16/19.
+//  Copyright © 2019 Nathanael Youngren. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifier = "Cell"
+
+class PhotosCollectionViewController: UICollectionViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.collectionView!.register(UICollectionViewCell.self, forCellWithReuseIdentifier: reuseIdentifier)
+        setTheme()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        setTheme()
+        collectionView.reloadData()
+    }
+
+    override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return photoController.photos.count
+    }
+
+    override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "PhotosCellID", for: indexPath)
+        guard let photoCell = cell as? PhotosCollectionViewCell else { return cell }
+    
+        let image = UIImage(data: photoController.photos[indexPath.item].imageData)
+        photoCell.imageView.image = image
+        photoCell.textLabel.text = photoController.photos[indexPath.item].title
```

Comment:
We don't need to set cell's `imageView` and `textLabel` here, because you already do it in the cell's `updateViews` function. And it's not a good choice, performance wise. We just need to pass related `photo` object. Delete line 35-36-37 and add following:
```
        let photo = photoController.photos[indexPath.item]
        photoCell.photo = photo
```